.travis.yml

```diff
@@ -5,6 +5,12 @@ python:
   - "3.4"
   - "3.5"
   - "3.6"
+
+jobs:
+  include:
+    - dist: xenial
```

Comment:
What's this bit about? Why can't 3.7 just get added to the list above?

Reply:
The default Travis image is still Trusty. Python 3.7 doesn't work on Trusty. https://github.com/travis-ci/travis-ci/issues/9069

Therefore 3.7 needs Xenial, but the current Travis Xenial image doesn't offer the older Python versions via default, so the old versions need to stay on the Trusty image for the moment.